android/src/main/java/com/reactnativecommunity/checkbox/ReactCheckBoxPackage.java

```diff
@@ -14,7 +14,7 @@
 public class ReactCheckBoxPackage implements ReactPackage {
     @Override
     public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
-      return Arrays.<NativeModule>asList(new RNCReactNativeCheckboxModule(reactContext));
+      return Collections.emptyList();
```

Comment:
What is the purpose of returning an empty Array instead of `List< NativeModule>`?

Reply:
Anyway, I will confirm it after merging